### ISSUE TITLE
There's no .gitignore in new projects

### ISSUE THREAD

**Issue by geealbers** [thegetty/quire#49]
**Steps to reproduce:**

1. Create a new Quire project with `quire new project-name`
2. Open the project in GitHub Desktop

**Expected behavior:**

User sees nothing in the list of changes. (Or maybe just `themes/default/package-lock.json` but I'd like to see us commit that in automatically after `quire install` is run.)

**Actual behavior:**

User sees a list of 30,230 changed files. One is themes/default/package-lock.json, the rest are in `themes/default/node_modules/`

<img width="1265" alt="Screen Shot 2021-03-04 at 11 12 31 AM" src="https://user-images.githubusercontent.com/7796401/110017885-8c645600-7cdb-11eb-8d3b-550f7f471b6b.png">

---

For users familiar with git and especially more technical users using it through the command line where the `themes/default/node_modules/` is listed as untracked, it would like be obvious that they wouldn't want to commit the node files in. For less technical users though and for those using GitHub Desktop, they might not know what to do and there's no easy way in the interface for them to untrack those files even if they knew to. And as those users start working, other kinds of files that shouldn't be tracked will also be introduced.

As part of the install process, **we should add a `.gitignore` file**. One used to be part of the standard install prior to the switch to the monorepo. https://github.com/thegetty/quire-starter/blob/master/.gitignore.

```
.DS_Store
.bundle
.cache
.sass-cache
public
site
node_modules
.tmp
```

I don't know how/if this one should be updated, but I would suggest we don't need to include `public` (the old name for the `site` folder), and **I wonder if we should include `site` at all. What's our best practice on that?** Do we recommend folks commit in the built files? On one hand it duplicates everything (like images) so could lead to a lot bloat. On the other hand, having the site files committed in would probably be better for long-term maintenance and potentially also deploying in some circumstances.

Regardless, I'd also suggest we take the opportunity to **remove the `site` files that are currently committed** to the default starter, as they're not complete and have caused some confusion (https://github.com/thegetty/quire-cli/issues/68 for example). I think it would be better if users had to run `quire site` themselves to generate all those files and thereby see how it works.

**Comment by geealbers** [comment]
Being tracked by Getty in Jira at: https://jira.getty.edu/browse/DEV-8259

**Comment by geealbers** [comment]
Closed with 5ef9acf
